internal/pkg/migrations/20250203145423_add_openstack_servers.tx.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS "openstack_server" (
```

Comment:
Schema for `openstack_server` does not seem correct when comparing it to the model.

A few issues noticed.

- Lots of fields have the `not null` constraint, while the model does not specify them as such
- Unique key is not correct as well when comparing it to the model.
- Also, unique key constraint does not follow the existing naming convention